### PR TITLE
Update DiCy

### PR DESCRIPTION
**Pull request by yitzchak**
DiCy v0.9.1 is more strict and adds option value validation. Update `Composer` as needed. Also update file extension filters to permit Pweave and literate Agda.

### CODE REVIEW

**Comment by yitzchak** [comment]
@thomasjo Can you take a look at this one?

**Comment by yitzchak** [comment]
Also resolves #342

File: lib/composer.js [hunk 1]
```diff
@@ -318,13 +322,12 @@ export default class Composer extends Disposable {
     latex.status.setIdle()
   }
 
-  async build (shouldRebuild) {
+  async build (shouldRebuild = false, silent = false) {
```
Comment (thomasjo) [inline lib/composer.js]: I think `quiet` is more frequently used, but what about making this more verbose and understandable; `suppressLogging`. Alternatively, flip the logic and name it `verbose` or some such.

File: lib/composer.js [hunk 2]
```diff
@@ -333,7 +336,7 @@ export default class Composer extends Disposable {
     }
 
     if (this.shouldUseDiCy()) {
-      return this.runDiCy(['load', 'build', 'log', 'save'], { ignoreCache: shouldRebuild },
+      return this.runDiCy(['load', 'build', 'log', 'save'], { ignoreCache: !!shouldRebuild },
```
Comment (thomasjo) [inline lib/composer.js]: This seems unnecessary given the argument has a default value of `false`.

Comment (yitzchak) [reply lib/composer.js]: Good point.

File: lib/composer.js (hunk 1 above)
Comment (yitzchak) [reply lib/composer.js]: What do think about `enableLogging`?

Comment (thomasjo) [reply lib/composer.js]: I like it :1st_place_medal:

**Pull request merged**